### DISCUSSION TITLE
Database queries

### DISCUSSION THREAD

**Discussion by agarwalrohit2503** [serverpod/serverpod#270] [Q&A] [answered]
After getting the tables.pgsql created using serverpod generate 
How can we run that in the pgsql to create the tables
Error:

![Screenshot 2022-09-09 at 2 31 04 PM](https://user-images.githubusercontent.com/33911927/189313413-8b17a273-d365-4e84-a5aa-166c3b0fcab6.png)

**Comment by vlidholt** [comment, marked as answer]
You will need to connect to the database using the credentials that you find in your `config/development.yaml` and `config/passwords.yaml` file using your favorite Postgres tool. I'm personally using Postico, but it's also possible to use the standard pgsql tool.

Then you can add or edit your tables from there.

Future versions of Serverpod will add built in tooling for managing the database and doing migrations.
